flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskCancelAsyncProducerConsumerITCase.java

```diff
@@ -150,10 +147,11 @@ public void testCancelAsyncProducerAndConsumer(@InjectMiniCluster MiniCluster fl
         }
 
         // Verify that async producer is in blocking request
-        assertTrue(
-                "Producer thread is not blocked: "
-                        + Arrays.toString(ASYNC_PRODUCER_THREAD.getStackTrace()),
-                producerBlocked);
+        assertThat(producerBlocked)
+                .withFailMessage(
```

Comment:
maybe use `as` too here, we can unify this

Reply:
fixed